### PR TITLE
Handlebars pages

### PR DESCRIPTION
**Pull request by danshoup**
Updated the newEvent handlebars page to include a datepicker instead of dropdown menus for all dates.  Datepicker is giving me a bit of trouple, but the format is here.

### CODE REVIEW

File: TemplateHTML/newEvent.html [hunk 1]
```diff
@@ -30,20 +30,19 @@
     <main class="container container-fluid mt-5">   
         <!-- Render the sub layout -->
         <div class="container d-flex justify-content-center">
-            
             <form>
```
Comment (bdstpierre) [inline TemplateHTML/newEvent.html]: This <form> tag will need a class so the javascript can attach a listener to the submit button.
<form class="new-event-form">

File: TemplateHTML/newEvent.html [hunk 2]
```diff
@@ -30,20 +30,19 @@
...
                     <p>Your first steps to creating your event begin with these few questions. You'll be on your way to securing new volunteers in no time!</p>
                     </div>
                 </div>
             
                 <div class="form-row">
-                  <div class="form-group">
-                    <label for="inputEmail4">Event Name</label>
-                    <input type="text" class="form-control" id="inputEventName4">
+                  <div class="form-group col-md-12">
+                    <label for="inputEvent">Event Name</label>
```
Comment (bdstpierre) [inline TemplateHTML/newEvent.html]: I am not 100% sure but I think <label for="some_name"> has to match <input id="some_name">

File: views/newEvent.handlebars [hunk 3]
```diff
@@ -1,154 +1,112 @@
...
+                <div class="row" style="margin-bottom: 30px;">
+                    <div class="twelve columns" style="text-align: center;">
+                    <h2>Create an Event</h2>
+                    <p>Your first steps to creating your event begin with these few questions. You'll be on your way to securing new volunteers in no time!</p>
+                    </div>
+                </div>
+            
+                <div class="form-row">
+                  <div class="form-group col-md-12">
+                    <label for="inputEvent">Event Name</label>
```
Comment (bdstpierre) [inline views/newEvent.handlebars:13]: I think the for="inputEvent" needs to match the id="inputEventName" on the line below (and it doesn't)

**Review by bdstpierre — COMMENTED** [review]
I commented in a couple of spots.

File: TemplateHTML/newEvent.html (hunk 1 above)
Comment (danshoup) [reply TemplateHTML/newEvent.html]: I think I fixed this in the recent push

File: TemplateHTML/newEvent.html (hunk 2 above)
Comment (danshoup) [reply TemplateHTML/newEvent.html]: Yes, I think you are right.  I believe I fixed that in the recent push also.

File: views/newEvent.handlebars (hunk 3 above)
Comment (danshoup) [reply views/newEvent.handlebars:13]: Not sure those need to match but I'm double checking the code right now

Comment (bdstpierre) [reply views/newEvent.handlebars:13]: I think this is what ties the label to the input.

**Review by bdstpierre — APPROVED** [review]

**Pull request merged**